package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vantiq-sdk",
-  "version": "1.1.4",
+  "version": "1.1.5",
```

Comment:
Since this is adding a new feature, the minor version should be updated -> 1.2.0
See convention stated in:  https://github.com/Vantiq/vantiq-sdk-java/pull/31#discussion_r878903541